app/screens/server/index.tsx

```diff
@@ -403,14 +411,21 @@ const Server = ({
                         disableServerUrl={disableServerUrl}
                         handleConnect={handleConnect}
                         handleDisplayNameTextChanged={handleDisplayNameTextChanged}
+                        handlePreauthSecretTextChanged={handlePreauthSecretTextChanged}
                         handleUrlTextChanged={handleUrlTextChanged}
                         keyboardAwareRef={keyboardAwareRef}
+                        preauthSecret={preauthSecret}
                         theme={theme}
                         url={url}
                         urlError={urlError}
                     />
                 </KeyboardAwareScrollView>
-                <AppVersion textStyle={styles.appInfo}/>
+                <View style={{height: 60, justifyContent: 'center', alignItems: 'flex-start', paddingLeft: 20}}>
```

Comment:
I'm not a big fan of having things so static in the styles (like height 60, or the padding 120 you have in the other style). We could reconsider what effect we are aiming for, and try to do a more dynamic thing.